document/paragraphproperties.go

```diff
@@ -199,3 +199,186 @@ func (p ParagraphProperties) SetHangingIndent(m measurement.Distance) {
 		p.x.Ind.HangingAttr.ST_UnsignedDecimalNumber = unioffice.Uint64(uint64(m / measurement.Twips))
 	}
 }
+
+// Bold returns true if paragraph font is bold.
+func (p ParagraphProperties) Bold() bool {
+	x := p.x.RPr
+	return getBool(x.B) || getBool(x.BCs)
+}
+
+// Italic returns true if paragraph font is italic.
+func (p ParagraphProperties) Italic() bool {
+	x := p.x.RPr
+	return getBool(x.I) || getBool(x.ICs)
+}
+
+// Caps returns true if paragraph font is capitalized.
+func (p ParagraphProperties) Caps() bool {
+	return getBool(p.x.RPr.Caps)
+}
+
+// Strike returns true if paragraph is striked.
+func (p ParagraphProperties) Strike() bool {
+	return getBool(p.x.RPr.Strike)
+}
+
+// DoubleStrike returns true if paragraph is double striked.
+func (p ParagraphProperties) DoubleStrike() bool {
+	return getBool(p.x.RPr.Dstrike)
+}
+
+// Outline returns true if paragraph outline is on.
+func (p ParagraphProperties) Outline() bool {
+	return getBool(p.x.RPr.Outline)
+}
+
+// Shadow returns true if paragraph shadow is on.
+func (p ParagraphProperties) Shadow() bool {
+	return getBool(p.x.RPr.Shadow)
+}
+
+// Emboss returns true if paragraph emboss is on.
+func (p ParagraphProperties) Emboss() bool {
+	return getBool(p.x.RPr.Emboss)
+}
+
+// RightToLeft returns true if paragraph text goes from right to left.
+func (p ParagraphProperties) RightToLeft() bool {
+	return getBool(p.x.RPr.Rtl)
+}
+
+// RStyle returns the name of paragraph runs style.
+func (p ParagraphProperties) RStyle() string {
+	if p.x.RPr.RStyle != nil {
+		return p.x.RPr.RStyle.ValAttr
+	}
+	return ""
+}
+
+// Font returns the name of paragraph font family.
+func (p ParagraphProperties) Font() string {
+	if fonts := p.x.RPr.RFonts; fonts != nil {
+		if fonts.AsciiAttr != nil {
+			return *fonts.AsciiAttr
+		} else if fonts.HAnsiAttr != nil {
+			return *fonts.HAnsiAttr
+		} else if fonts.CsAttr != nil {
+			return *fonts.CsAttr
+		}
+	}
+	return ""
+}
+
+// EastAsiaFont returns the name of paragraph font family for East Asia.
+func (p ParagraphProperties) EastAsiaFont() string {
+	if fonts := p.x.RPr.RFonts; fonts != nil {
+		if fonts.EastAsiaAttr != nil {
+			return *fonts.EastAsiaAttr
+		}
+	}
+	return ""
+}
+
+// ColorStr returns the hex value of paragraph color.
+func (p ParagraphProperties) ColorStr() string {
```

Comment:
Consistency, probably also call this Color() ?  or HexColor.  Should be named consistently.
There is another similar function this PR.

Reply:
Renamed to HexColor because there is already a function with name Color and it returns color in the form of x *wml.CT_Color.